modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -688,8 +688,20 @@ func (h Handler) prepareRequest(req *http.Request, repl *caddy.Replacer) (*http.
 		req.Header.Set("Upgrade", reqUpType)
 	}
 
+	// Set up the PROXY protocol info
+	// TODO: We should probably migrate away from netip, but due to
```

Comment:
Why migrate away from it?

Reply:
Oops, that was meant to say `net.IP`. But also this comment is sorta in the wrong place. The PROXY protocol lib is using the older `net.IP` interfaces instead of the newer `netip.Addr`, that's what this was meant to point out.